test/tutorial/step-7-testing-post-endpoints.md

```diff
@@ -0,0 +1,28 @@
+### Testing POST endpoints
+
+If we remember in our Node-Girls workshop, the `POST` request is quite different to a `GET` request.
```

Comment:
I'd call it the "Express workshop" or "CMS workshop", as this would be a Node Girls workshop too!

Reply:
All done - I renamed it to "Express Workshop" 😄